plugins/module_utils/utils.py

```diff
@@ -96,10 +96,11 @@ def recursive_replace(data, path, keys, new_value):
         if len(keys) == 1:
             # Update the existing configuration
             if new_value is not None and data.get(key) != new_value:
```

Comment:
Shall we change the name of this function to recursive_add_repalce() or something more appropriate?